### DISCUSSION TITLE
Is there any way to use colSpan in Foote

### DISCUSSION THREAD

**Discussion by dnjsgur0629** [KevinVandy/material-react-table#269] [Q&A] [answered]
Is there a way to represent using colSpan in Footer as shown below?
![image](https://user-images.githubusercontent.com/65391273/208347367-0d24dbd9-0e4c-4cae-90fa-a3d05b1529e8.png)

Of course, I can change colSpan in muiTableFooterCellProps, but then the column will be pushed as shown below.
![image](https://user-images.githubusercontent.com/65391273/208349235-7c5b930b-2737-4977-9a78-1afe9a5d563a.png)

If there is no way, is there an alternative?

**Comment by KevinVandy** [comment, marked as answer]
Manual colSpans and rowSpans just are not easily supported in TanStack Table right now, so I don't really have an answer for this yet.
